### PR TITLE
Add simple script for benchmark execution

### PR DESCRIPTION
**Pull request by Vyacheslav-Smirnov**
- Separate script
 - Use files from Intel Anaconda channel
 - Add modules list as parameter

### CODE REVIEW

File: buildscripts/run_benchmarks.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+def run_benchmarks(sdc_utils, module_list, num_threads_list):
+    os.chdir(str(sdc_utils.src_path.parent))
+
+    for module in module_list:
+        for num_threads in num_threads_list:
+            os.environ['NUMBA_NUM_THREADS'] = num_threads
+            sdc_utils.log_info(f'Run Intel SDC benchmarks on {num_threads} threads', separate=True)
+            run_command(f'python -W ignore -m sdc.runtests {module}')
```
Comment (densmirn) [inline buildscripts/run_benchmarks.py]: Where the function is imported from?

File: buildscripts/run_benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+from pathlib import Path
+from utilities import SDC_Build_Utilities
+
+
+def run_benchmarks(sdc_utils, module_list, num_threads_list):
+    os.chdir(str(sdc_utils.src_path.parent))
+
+    for module in module_list:
+        for num_threads in num_threads_list:
```
Comment (densmirn) [inline buildscripts/run_benchmarks.py]: You could combine the loops using `itertools.product`.

File: buildscripts/run_benchmarks.py (hunk 1 above)
Comment (Vyacheslav-Smirnov) [reply buildscripts/run_benchmarks.py]: It is a bug, going to update it

File: buildscripts/run_benchmarks.py (hunk 2 above)
Comment (Vyacheslav-Smirnov) [reply buildscripts/run_benchmarks.py]: To my mind 2 loops here are more obvious than using itertools.product

File: buildscripts/run_benchmarks.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            os.environ['NUMBA_NUM_THREADS'] = num_threads
+            sdc_utils.log_info(f'Run Intel SDC benchmarks on {num_threads} threads', separate=True)
+            sdc_utils.run_command(f'python -W ignore -m sdc.runtests -v {module}')
```
Comment (PokhodenkoSA) [inline buildscripts/run_benchmarks.py]: Actually `module` could be any parameters for Python unittest module. I usually filter my tests like: `sdc.tests.test_series -k series_max`

Comment (Vyacheslav-Smirnov) [reply buildscripts/run_benchmarks.py]: Then I will add a separate parameter for "args"

**Pull request merged**